buildSrc/src/main/java/Dependencies.kt

```diff
@@ -28,6 +28,9 @@ object Versions {
 
     const val material = "1.0.0"
 
+    const val firebase_core = "16.0.1"
```

Comment:
why are these different versions? I'm assuming that's necessary for something, but it's a little surprising. Can you add a comment clarifying this?

Reply:
The core library is separate from the messaging one, as we add more firebase features, each one has their own dependency version. I'm not sure what the value of adding a comment to this would be. How about a "firebase" object similar to how we've done it for AndroidX?